docs/managing-dragonfly/flags.md

```diff
@@ -0,0 +1,551 @@
+# Command line arguments(flags)
+
+Dragonfly can be tuned and configured by a set of config flags. These flags can be:
+
+1. Passed to a Dragonfly instance directly at startup. E.g, `./dragonfly --port=6379`.
+2. Loaded from a file by a Dragonfly instance. E.g. `./dragonfly --flagfile=path_to_flags/flags.txt`
+3. Can be set via env variables by adding the prefix `DFLY` followed by the flag name. E.g. `export DFLY_port=6379` (note it's case sensitive)
+4. At runtime via `CONFIG SET` command. Not all flags, can be configured at runtime.
+
+You can try `./dragonfly --helpfull` to get a list of all flags or `--help=substring` shows help for
+flags which include specified substring in either in the name, description or path.
+
+## Available flags
+
+### `--use_set2`
```

Comment:
Let's either sort the flags alphabetically or by importance, wdyt?

I'd vote to put port, cache_mode, cluster_mode, maxmemory, dbnum, bind, requirepass, dbfilename, snapshot_cron first (not necessarily in this order)

Reply:
I will put the important one first and let the rest follow - I do not want to sort a huge list of flags :P (maybe chat gpt could this nicely ? :eyes: